x-pack/plugins/monitoring/server/lib/elasticsearch/nodes/get_nodes/map_nodes_info.js

```diff
@@ -20,22 +20,22 @@ export function mapNodesInfo(nodeHits, clusterStats, shardStats) {
   const clusterState = get(clusterStats, 'cluster_state', { nodes: {} });
 
   return nodeHits.reduce((prev, node) => {
-    const sourceNode = get(node, '_source.source_node');
+    const sourceNode = get(node, '_source.node_stats');
 
     const calculatedNodeType = calculateNodeType(sourceNode, get(clusterState, 'master_node'));
     const { nodeType, nodeTypeLabel, nodeTypeClass } = getNodeTypeClassLabel(sourceNode, calculatedNodeType);
-    const isOnline = !isUndefined(get(clusterState, [ 'nodes', sourceNode.uuid ]));
+    const isOnline = !isUndefined(get(clusterState, [ 'nodes', sourceNode.node_id ]));
 
     return {
       ...prev,
-      [sourceNode.uuid]: {
-        name: sourceNode.name,
+      [sourceNode.node_id]: {
+        name: sourceNode.node_id,
```

Comment:
Pinging @ycombinator. Do we have a replacement plan for this? I can't remember from our earlier discussions, but are we able to inject the node's `name` (whatever node the beat is talking to) into the document shipped by MB? Or what was our plan for this?